src/shared_memory_manager.cc

```diff
@@ -365,6 +405,18 @@ SharedMemoryManager::RegisterCUDASharedMemory(
   // Get CUDA shared memory base address
   TRITONSERVER_Error* err =
       OpenCudaIPCRegion(cuda_shm_handle, &mapped_addr, device_id);
+  if (err != nullptr) {
```

Comment:
Is this a duplicate check to the err check on line 420 below?

Reply:
Good catch! Removed the duplicate.